signing_service/signing_service/tests/tests.py

```diff
@@ -107,14 +118,68 @@ def test_that_signing_service_should_reraise_different_socket_erros(self):
         mock_socket_close.assert_called_once()
 
 
+class SigningServiceHandleConnectionTestCase(TestCase):
+
+    def setUp(self):
+        self.host = '127.0.0.1'
+        self.port = 8000
+        self.initial_reconnect_delay = 2
+        # TODO: Use predefined ones when SigningService will accept Concent private key as parameter.
+        self.concent_public_key = b'\xf3\x97\x19\xcdX\xda\x86tiP\x1c&\xd39M\x9e\xa4\xddb\x89\xb5,]O\xd5cR\x84\xb85\xed\xc9\xa17e,\xb2s\xeb\n1\xcaN.l\xba\xc3\xb7\xc2\xba\xff\xabN\xde\xb3\x0b\xa6l\xbf6o\x81\xe0;'
+
+    def test_that__handle_connection_should_send_wrapped_service_refuse_if_frame_is_invalid(self):
+        # Prepare message with wrong signature
+        middleman_message = GolemMessageMiddlemanMessage(Ping(), 99)
+        raw_message = middleman_message.serialize(
+            # TODO: Use predefined ones when SigningService will accept Concent private key as parameter.
+            private_key=b'l\xcdh\x19\xeb$>\xbcG\xa1\xc7v\xe8\xd7o\x0c\xbf\x0e\x0fM\x89lw\x1e\xd7K\xd6Hnv$\xa2',
```

Comment:
Also please use `CONCENT_PRIVATE_KEY` which you generated on the top of this file

Reply:
Same here.